modules/services/pbgopy.nix

```diff
@@ -5,18 +5,41 @@ with lib;
 let
   cfg = config.services.pbgopy;
   package = pkgs.pbgopy;
+
+  commandLine = concatStringsSep " " ([ "${package}/bin/pbgopy serve" ]
+    ++ optional (cfg.port != 9090) "--port ${toString cfg.port}"
+    ++ optional (cfg.cache.ttl != "24h") "--ttl ${cfg.cache.ttl}"
```

Comment:
Is 24 hour the current default cache time?

Reply:
It is.

Reply:
I would be in favor of explicitly setting the `port` and `cache.ttl` arguments instead of using `optional`. Since we have the `default` values already in the options, doing this is kinda pointless IMO. Additionally, if upstream ever changes the defaults, then we only have one place to change it (in the options config below) rather than two places.

Reply:
I agree. @IvarWithoutBones would it be possible to make the mentioned changes?